NonModernCMake.md

```diff
@@ -0,0 +1,82 @@
+# Awesome CMake [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
+
+[<img src="https://rawgit.com/onqtam/awesome-cmake/master/cmake-logo.svg" align="right" width="100">](https://cmake.org/)
+
+> This is an archive of pre-modern [CMake](https://cmake.org/) scripts, modules, examples and others
```

Comment:
Maybe use a more concrete definition? Something like "pre-v3" for example.

It would also be nice to mention the criteria you listed in the PR description (usage of `include_directories`, etc.).

Reply:
pre-v3 isn't really non modern, 2.8 had some modern (ie target_*) things imo.

The commit actually has the PR description as description :)

https://github.com/onqtam/awesome-cmake/commit/280a93e29f7b4ba1e1c73a251b84c15784ff49f5

Reply:
Oh maybe you meant in the `.md` file?

Reply:
> Oh maybe you meant in the `.md` file?

Yes :)